File: API/APIMeow/Models/Recorrencia.cs

using System.ComponentModel.DataAnnotations;

public class Recorrencia
{
   [Key]
   public int IdRecorrencia { get; set; }
   public int QtsDias { get; set; }
   public int QtsMeses { get; set; }
   public int QtsAnos { get; set; }
}

File: front_meow/lib/models/recorrencia.dart
class Recorrencia {
  final int idRecorrencia;
  final int qtosDias;
  final int qtosMeses;
  final int qtosAnos;

  Recorrencia({
    required this.idRecorrencia,
    required this.qtosDias,
    required this.qtosMeses,
    required this.qtosAnos,
  });

  factory Recorrencia.fromJson(Map<String, dynamic> json) {
    return Recorrencia(
      idRecorrencia: json['IdRecorrencia'] ?? json['idRecorrencia'],
      qtosDias: json['QtsDias'] ?? json['qtosDias'] ?? json['qtosDias'],
      qtosMeses: json['QtsMeses'] ?? json['qtosMeses'] ?? json['qtosMeses'],
      qtosAnos: json['QtsAnos'] ?? json['qtosAnos'] ?? json['qtosAnos'],
    );
  }
}
